src/commands/cronjob/create.tsx

```diff
@@ -4,65 +4,49 @@ import {
   processFlags,
 } from "../../rendering/process/process_flags.js";
 import { ReactNode } from "react";
-import { Flags } from "@oclif/core";
 import { assertStatus } from "@mittwald/api-client-commons";
 import { Success } from "../../rendering/react/components/Success.js";
 import { Value } from "../../rendering/react/components/Value.js";
 import { appInstallationFlags } from "../../lib/resources/app/flags.js";
-import Duration from "../../lib/units/Duration.js";
+import { cronjobFlagDefinitions } from "../../lib/resources/cronjob/flags.js";
+import { buildCronjobDestination } from "../../lib/resources/cronjob/destination.js";
+
+import type { MittwaldAPIV2Client } from "@mittwald/api-client";
+import { Flags } from "@oclif/core";
 
 type Result = {
   cronjobId: string;
 };
 
+type CronjobCreationData = Parameters<
+  MittwaldAPIV2Client["cronjob"]["createCronjob"]
+>[0]["data"];
+
 export class Create extends ExecRenderBaseCommand<typeof Create, Result> {
   static summary = "Create a new cron job";
   static flags = {
     ...appInstallationFlags,
     ...processFlags,
-    description: Flags.string({
-      required: true,
-      summary: "Description of the cron job",
-      default: undefined,
-    }),
-    interval: Flags.string({
-      required: true,
-      summary: "Interval of the cron job, in standard UNIX cron syntax",
-      default: undefined,
+    description: cronjobFlagDefinitions.description({ required: true }),
+    interval: cronjobFlagDefinitions.interval({ required: true }),
+    email: cronjobFlagDefinitions.email(),
+    timeout: cronjobFlagDefinitions.timeout({ required: true }),
```

Comment:
Changing the `--timeout` flag from a "duration" flag to a "number" flag is a UX downgrade, IMO. Before, you could specify values like `1h`, now you need to know how many seconds an hour has. Also, I think I would prefer having a (sensible) default value here, instead of this field being required.

Reply:
I couldnt figure out how to implement this with the centralized flags.
`timeout: Flags.custom<Duration.relativeFlag>()` seems nonsensical.
Seem to be "too custom for custom flags"
So I rolled that flag back to how it worked before since you're abeolutely right, the new implementation was a downgrade in every regard.